example/lib/line_chart/samples/line_chart_sample1.dart

```diff
@@ -41,7 +41,14 @@ class _LineChart extends StatelessWidget {
   LineTouchData get lineTouchData1 => LineTouchData(
         handleBuiltInTouches: true,
         touchTooltipData: LineTouchTooltipData(
-          tooltipBgColor: Colors.blueGrey.withOpacity(0.8),
+          tooltipBgGradient: const LinearGradient(
```

Comment:
This gradient doesn't fit with the line chart sample1 design theme.
Can you please make it look better, or use it on another sample?

<img width="335" alt="image" src="https://user-images.githubusercontent.com/7009300/159502798-3b8a3fc1-be06-4122-b57a-6468a071b241.png">